installation_and_upgrade/Jenkinsfile_vhd_creation

```diff
@@ -17,6 +17,7 @@ pipeline {
     stage("Checkout") {
       steps {
         timeout(time: 100, unit: 'MINUTES') {
+          bat "echo %USERNAME%"
```

Comment:
Is this debug code or useful in general?

Reply:
It is not and will be removed after I've finished testing (I've made PRs before finishing the ticket so haven't fully cleaned them up yet)